health/health.c

```diff
@@ -344,7 +344,9 @@ static inline void health_alarm_execute(RRDHOST *host, ALARM_ENTRY *ae) {
         }
     }
 
-    snprintfz(command_to_run, ALARM_EXEC_COMMAND_LENGTH, "exec %s '%s' '%s' '%u' '%u' '%u' '%lu' '%s' '%s' '%s' '%s' '%s' '" CALCULATED_NUMBER_FORMAT_ZERO "' '" CALCULATED_NUMBER_FORMAT_ZERO "' '%s' '%u' '%u' '%s' '%s' '%s' '%s' '%s' '%s' '%d' '%d' '%s' '%s'",
+    char *edit_command = health_edit_command_from_source(ae->source?ae->source:"UNKNOWN");
```

Comment:
Hello @MrZammler ,

Here the `UNKNOWN` is 100% in uppercase, but at like 377, we have `Unknown`. Is there a special motive for this difference?

Reply:
I saw the motive in the next files, another doubt, instead to compare with `UNKNOWN`, why did you not give a `NULL` pointer?

Reply:
Well, for the class attribute, the `Unknown` has been decided, however the source attribute is already sending `UNKNOWN`, so it will keep like this...

In any case, I will merge this PR to the branch (not the master), so the rest of the team can test the changes. Before this goes to master we will have another chance to review it.

Thank you Thiago!